api/resolvers/item.js

```diff
@@ -167,7 +167,7 @@ const relationClause = (type) => {
       clause += ' FROM "Item" '
   }
 
-  return clause
+  return clause + ' LEFT JOIN "Sub" ON "Sub"."name" = "Item"."subName" '
```

Comment:
Comments don't have `subName` set. They defer to their root for the `subName`. e.g. when I go to `/recent/comments` it's empty because no comments match the new nsfw conditions. 

So you'll want to join on `root.subName` as in the above cases where `root` is joined.

I'm sorry, I know this item query stuff is hairy.

Reply:
https://github.com/stackernews/stacker.news/pull/788/commits/9887ec5ef267d0bcce05b8730ddd6985c5673423

whoa i didn't realize comments were also items 🤯 but that makes so much sense. good reminder to add comments in my local data.